### PR TITLE
Bug: Flash messages should be name spaced

### PR DESCRIPTION
**Pull request by EdwinRozario**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,11 +61,21 @@ class ApplicationController < ActionController::Base
 end
 ```
 
-Now you should be able to go to http://localhost:3000/safety_cone and manage your routes.
-
-
-SafetyCone uses flash messages. It is expected that flash messages are rendered on all views.
+In View
+
+```
```
Comment (Reinkaos) [inline README.md]:
```html
<div>Hi Dan!</div>
```

File: README.md [hunk 2]
```diff
@@ -61,11 +61,21 @@ class ApplicationController < ActionController::Base
...
 ```
 
-Now you should be able to go to http://localhost:3000/safety_cone and manage your routes.
-
-
-SafetyCone uses flash messages. It is expected that flash messages are rendered on all views.
+In View
+
+``` 
+<div class="container">
```
Comment (richardmatthewsdev) [inline README.md]: add html after the ``` for prettyness

**Comment by richardmatthewsdev** [comment]
Add some specs for your newly named flash message :)

File: lib/safety_cone/filter.rb [hunk 3]
```diff
@@ -12,12 +12,14 @@ def self.included(base)
     # based on the configuration.
     def safety_cone_filter
       if cone = fetch_cone
-        flash.clear
-        flash[notice_type(cone.measure)] = cone.message
+        flash[:safetycone_notice] = flash[:safetycone_alert] = nil
+        flash["safetycone_#{notice_type(cone.measure)}"] = cone.message
         redirect_to safety_redirect if cone.measure == 'block'
```
Comment (richardmatthewsdev) [inline lib/safety_cone/filter.rb]: Have a think about the name 'measure' as it is misleading. I would think something like type would be good.

**Pull request merged**